dotnet-logging/Log4netAppenders/Tests/Log4netAppendersTests.cs

```diff
@@ -0,0 +1,33 @@
+using log4net.Config;
+
+namespace Tests
+{
+public class Log4netAppendersTests : IClassFixture<WebApplicationFactory<Program>>
```

Comment:
There's some odd indentation happening here. We could remove the `{}` around the `namespace` declaration to resolve this. Or indent the class and methods.